pkg/scheduler/scheduler.go

```diff
@@ -256,6 +256,7 @@ func (s *Scheduler) admit(ctx context.Context, e *entry) error {
 	}
 	newWorkload.Spec.Admission = admission
 	if err := s.cache.AssumeWorkload(newWorkload); err != nil {
```

Comment:
If there is an error, `AssumeWorkload` doesn't store anything. So this change is not doing anything. Unless I'm missing something?